app/views/token_packs/_token_pack_form.html.slim

```diff
@@ -0,0 +1,32 @@
+.d-flex.justify-content-center
+  .card.token-card.col-lg-5.col-md-7.col-sm-9.col-11
+    .card-header
+      h5.card-title
+        - if action_name == "edit"
+          = "Edit Token Pack"
+        - else
+          = "New Token Pack"
+    .card-body
+      = form_with model: token_pack do |f|
+        .form-group
+          = f.label :title, class: "form-label"
+          = f.text_field :title, class: "form-control"
+        .form-group
+          = f.label :description, class: "form-label"
+          = f.text_field :description, class: "form-control"
+        .form-group
+          .row
+            .col
+              .form-group
+                = f.label :price, class: "form-label"
+                = f.number_field :price, class: "form-control", step: 0.01
+            .col
+              .form-group
+                = f.label :token_amount, class: "form-label"
+                = f.number_field :token_amount, class: "form-control", step: 1
+        .card-footer
+          .form-group
+            - if action_name == "edit"
```

Comment:
Šeit sanak mazliet viltigak jāuztaisa. Sanak, kad nostrāda validacija tev jau būs action Update, bet rederesis šis pats skats. 

Pamegini vispar izmantot token_pack.new_record?
`
= f.submit class: "btn btn-primary", value: token_pack.new_record? ? 'Create' : 'Update'`

Augstak tev ari jāpielabo condition